server/src/ResolveLocation.c

```diff
@@ -0,0 +1,114 @@
+/*
+ * kalman.c
+ *
+ *  Created on: 23 nov 2017
+ *      Author: alexs
+ */
```

Comment:
Remove comment header, it looks outdated anyway :smiley:

Reply:
the whole file is actually. I will remove this.